Gemfile

```diff
@@ -0,0 +1,34 @@
+source "https://rubygems.org"
+
+gem "bootsnap", require: false
+gem "importmap-rails"
+gem "jbuilder"
+gem "kamal", require: false
+gem "pg", "~> 1.1"
+gem "propshaft"
+gem "puma", ">= 5.0"
+gem "rails", "~> 8.0.1"
+gem "solid_cable"
+gem "solid_cache"
+gem "solid_queue"
+gem "stimulus-rails"
+gem "tailwindcss-rails"
+gem "thruster", require: false
+gem "turbo-rails"
+gem "tzinfo-data", platforms: %i[ windows jruby ]
+
+group :development, :test do
+  gem "dotenv-rails"
+  gem "brakeman", require: false
+  gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
+  gem "rubocop-rails-omakase", require: false
+end
+
+group :development do
+  gem "web-console"
+end
+
+group :test do
+  gem "capybara"
+  gem "selenium-webdriver"
```

Comment:
Do we need these gems for test purposes? If not, let's remove them

Reply:
Fixed